### ISSUE TITLE
Figure out why datadog handler doesn't see success anymore

### ISSUE THREAD

**Issue by jaymzh** [socallinuxexpo/scale-chef#152]
It sees failures, but not success... seems like a problem in the handler itself.

**Comment by irabinovitch** [comment]
I think the alert might be getting misinterpreted. Currently as defined the monitor triggers an alert if a chef run fails more than twice in 1 hour  for a given role.  Right now scale-web1 and scale-web1-test both have the role "web".  scale-web1 is passing its chef runs.  scale-web1-test is failing on an SSL cert being missing.  

I've muted the alert for now.

In terms of next steps we need to:
1) Fix the web-test and lists-test hosts so their runs pass.
2) we can modify the alert to be clearer.  We can either change the query to group by host (or another tag we thing is more clear). Or we can use different roles for test vs prod.

**Comment by jaymzh** [comment]
* scale-lists-test1 is fixed.
* scale-reg-test1 was totally fucked and can not have possibly ever run chef before unless it was rebuilt from scratch recently. It had no chef install, no git install, no password on it, no keys on it, nothing. I had to bootstrap it manually and then run Chef. I got it to run Chef, but it lacks certs, I'll need to setup LE test certs on it, which means it needs a real honest to god public hostname (or we can just kill that host if you prefer). Your call.
* scale-web-test1 is fine, minus the same cert issue affecting scale-reg-test1

As for monitoring, it doesn't makes sense - at our scale of 1 prod host each - to monitor "by role" - no one needs to get paged if a test host breaks but someone should DEFINITELY get yelled at if a prod host is down. Lets just change it to per-host.

I'll fix the test hosts if you give them proper DNS names (test-reg.socallinuexpo.org et. al), I still don't think I have access to our DNS infra (or if I do, point me at it and I'll do that). Or we can kill them.
